docs/tutorial.md

```diff
@@ -0,0 +1,721 @@
+         chunk    {0} index.html 541 kB [entry] [rendered]
+             [0] ./~/lodash/lodash.js 540 kB {0} [built]
+             [1] ./~/html-webpack-plugin/lib/loader.js!./src/index.html 533 bytes {0} [built]
+             [2] (webpack)/buildin/global.js 509 bytes {0} [built]
+             [3] (webpack)/buildin/module.js 517 bytes {0} [built]
+     webpack: Compiled successfully.
+     ```
+
+   The system opens a browser displaying your running application.
+
+  ![](images/blockstack-signin.png)
+
+   At this point, the browser is running a Blockstack server on your local host.
+   This is for testing your applications only.
+
+6. Choose **Sign in with Blockstack**
+
+   The system displays a prompt allowing you to create a new Blockstack ID or restore an existing one.
+
+    ![](images/create-restore.png)
+
+7. Follow the prompts appropriate to your situation.
+
+    If you are restoring an existing ID, you may see a prompt about your user
+    being nameless, ignore it. At this point you have only a single application
+    on your test server.  So, you should see this single application, with your
+    own `blockstack.id` display name, once you are signed in:
+
+    ![](images/running-app.png)
+
+
+### Add a redirect end point to your application
+
+When a user opens the webapp from the Blockstack browser on an iOS phone,
+you want the web app to redirect the user to your iOS application. The work
+you do here will allow it.
+
+1. From the terminal command line, change directory to the root of your web
+application directory.
+
+2. Use the `touch` command to add a redirect endpoint to your application.
+
+   This endpoint on the web version of your app will redirect iOS users back
+   to your mobile app.
+
+    ```bash
+    $ touch public/redirect.html
+    ```
+
+3. Open `redirect.html` and add code to the endpoint.
+
+    ```
+    <!DOCTYPE html>
+    <html>
+    <head>
+      <title>Hello, Blockstack!</title>
+      <script>
+      function getParameterByName(name) {
+        var match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
+        return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
+      }
+
+      var authResponse = getParameterByName('authResponse')
+      window.location="myblockstackapp:" + authResponse
+      </script>
+    <body>
+    </body>
+    </html>
+    ```
+
+    Blockstack apps are identified by their domain names.  The endpoint will
+    receive a get request with the query parameter `authResponse=XXXX` and
+    should redirect the browser to `myblockstackapp:XXXX`.
+
+    `myblockstackapp:` is custom protocol handler. The handler should be unique
+    to your application. Your app's web-based authentication uses this handler
+    to redirect the user back to your iOS app. Later, you'll add a reference
+    to this handler in your iOS application.
+
+5. Close and save the `redirect.html` file.
+6. Ensure your Blockstack compiles successfully.
```

Comment:
Typo: "Blockstack app"

Also, would be good to expand as "Ensure your Blockstack app compiles successfully with `npm start`"

Reply:
👍